addons/payment_stripe/models/payment_acquirer.py

```diff
@@ -13,6 +13,8 @@ class PaymentAcquirerStripe(models.Model):
 
     stripe_secret_key = fields.Char(required_if_provider='stripe')
     stripe_publishable_key = fields.Char(required_if_provider='stripe')
+    image_url = fields.Char("Image URL", required_if_provider='stripe',
```

Comment:
I wanted to use the logo as well but Stripe has a requirement for a **square** image, company logo is most of the time a different aspect ratio. Also there are a number of other messages that can be sent from Stripe website/api (like receipts, manual payments, etc) and is probably a good idea of reusing the image set in Stripe profile, that way the customer will have the same experience when processing payments from Odoo or when renewing card info using Stripe web site or on email confirmations received from Stripe directly.